### PR TITLE
Add license info of libil, cc #5500

### PR DESCRIPTION
**Pull request by clairetsai818**
https://github.com/lynxluna/libil/blob/master/README.md#license

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

File: package.json [hunk 1]
```diff
@@ -36,5 +36,6 @@
   },
   "engines": {
     "node": "^4.2.3"
-  }
+  },
+  "license": ["MIT", "public domain"]
```
Comment (PeterDaveHello) [inline package.json]: This is unrelated to this PR.
